sw/device/lib/testing/BUILD

```diff
@@ -461,3 +461,15 @@ cc_library(
         "//sw/device/lib/testing/test_framework:check",
     ],
 )
+
+cc_library(
+    name = "usb_logging",
```

Comment:
The UART and SPI_DEVICE logging code is part of the OTTF (in `ottf_console.{c,h}`, see https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_framework/ottf_console.h). Since there is a lot more code required to log over USB it looks like, perhaps this should go in `ottf_usb_console.{c,h}` and then get integrated into the OTTF console library?

Reply:
(ok to do this in a follow up too)

Reply:
no problem, just wanted to point out I think there is a path towards future OTTF integration of this feature, and would probably be nice to have IMO, but we can cross that bridge later, and I am happy to help when the time comes. Thanks for adding this capability @alees24 !